src/test/java/core/basesyntax/services/WriteServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.WriteServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class WriteServiceImplTest {
+    private static final String PATH_TO_FILE = "src/test/java/resources/report.csv";
+    private static final String INVALID_PATH = "invalid/path/to/file";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
```

Comment:
same

Reply:
I still do not understand why constants do not have a good tone here

Reply:
answered above